src/Monitoring/monitoring.ts

```diff
@@ -0,0 +1,30 @@
+import express from "express";
+import client from "prom-client";
+
+class Monitoring {
```

Comment:
Let's change the folder name to lowercase please, so in that case, you may rename the file name as well to `monitoring/index.ts`

Reply:
Done with the last commit as "monitoring/index.ts".